app/assets/javascripts/components/StickyColumn.js

```diff
@@ -148,13 +148,21 @@ define(['jquery', 'DoughBaseComponent', 'eventsWithPromises'], function($, Dough
       }
     }
 
-    if (this.isAtBottom && scrollTop < this.bottom) {
+    if (this.isAtBottom && scrollTop < this.bottom && scrollTop > this.top) {
```

Comment:
Do you think nesting these two `if (this.isAtBottom)` conditions in a single one would make it easier to read? I.e. how you've done it for `if (this.isFixed)` here https://github.com/moneyadviceservice/frontend/blob/sticky_column_back_top_fix/app/assets/javascripts/components/StickyColumn.js#L134